test/functional/p2p_addr_relay.py

```diff
@@ -184,7 +198,68 @@ def relay_tests(self):
 
         self.nodes[0].disconnect_p2ps()
 
+    def sum_addr_messages(self, msgs_dict):
+        val = 0
+        for msg in ['addr', 'addrv2', 'getaddr', 'sendaddrv2']:
+            if msg in msgs_dict:
+                val += msgs_dict[msg]
+        return val
```

Comment:
```suggestion
        return sum(bytes_recvd for (msg, bytes_recvd) in msgs_dict.items() if msg in ['addr', 'addrv2', 'getaddr', 'sendaddrv2'])
```

Reply:
Also, remove `sendaddrv2` from the list?

Reply:
done and done